handbook/engineering/security/index.md

```diff
@@ -89,11 +89,31 @@ On the security team, we work by planning, tracking, and reviewing - creating a
    
 ### Learning
 
-After the each release, we hold a [retrospective](https://about.sourcegraph.com/retrospectives). We try to understand the degree to which we achieved the goals we communicaed at the beginning of the iteration.  We idenitfy what went well and what our opportunities for imporvement.  We actively choose one of the things we've learned, and target its improvement.
+After the each release, we hold a [retrospective](https://about.sourcegraph.com/retrospectives). We try to understand the degree to which we achieved the goals we communicated at the beginning of the iteration.  We identify what went well and what our opportunities for improvement.  We actively choose one of the things we've learned, and target its improvement.
+
+### Working with other teams
+
+The security team helps other teams investigate, orient, review, and sometimes plan changes. 
+
+**Small changes**
+
+Small changes, defined as less than 1 day of effort are communicated in a GitHub issue, either in the [sourcegraph repository](https://www.github.com/sourcegraph/sourcegraph) or, when sensitive in nature, the [security repository](https://www.github.com/sourcegraph/security-issues).  We provide a detailed bug report, including the steps to reproduce, and an example of the desired state.  Where possible, we identify and propose a fix, and submit a pull request to the [code owner](../code_reviews.md#who-should-i-get-a-code-review-from).
+
+**Medium changes**
+
+Medium changes operate on the same principle as small changes, with the exception of the team not submitting code changes. These changes also include a documented test plan for validation of the security change. 
+
+**Large changes**
```

Comment:
Where is the approximate line between medium and large? Medium is a few days, large is over a week?

Reply:
That feels about right.  Large is also based on scope such as cross cutting changes. Updated language.